src/Look/InfoBoxLook.cpp

```diff
@@ -73,10 +74,11 @@ InfoBoxLook::Initialise(bool _inverse, bool use_colors,
 }
 
 void
-InfoBoxLook::ReinitialiseLayout(unsigned width)
+InfoBoxLook::ReinitialiseLayout(unsigned width, unsigned scale_title_font)
 {
   FontDescription title_font_d(8);
-  AutoSizeFont(title_font_d, width, _T("123456789012345"));
+  unsigned scaled_width = (width * Clamp(scale_title_font,50U,150U)) / 100;
```

Comment:
What's the point of clamping here?

Reply:
remove Clamp since scale_title_font is clamped already.